weasyprint/css/__init__.py

```diff
@@ -273,18 +273,18 @@ def set_computed_styles(cascaded_styles, computed_styles,
     declaration priority (ie. ``!important``) and selector specificity.
 
     """
-    if parent is None:
-        parent_style = None
-    else:
-        parent_style = computed_styles[parent, None]
-
+    parent_style = computed_styles[parent, None] \
+        if parent is not None else None
+    root_style = computed_styles.get((root, None), {})
```

Comment:
`.get()` here should be `[]` indexing. We want this to fail early if the root element’s style can’t be found. This should be fine since `get_all_computed_styles` deals with the root element first.

But this brings the question: what about computing styles for the root element itself? Let’s look at the spec:

https://drafts.csswg.org/css-values/#rem
> Equal to the computed value of font-size on the root element. When specified on the font-size property of the root element, the rem units refer to the property’s initial value.

All in all, please replace this line with:

```python
root_style = computed_styles[root, None] if root is not element else properties.INITIAL_VALUES
```